vignettes/bds_finding.Rmd

```diff
@@ -522,14 +607,19 @@ advs <- derive_extreme_flag(
   mode = "last",
   filter = !is.na(AVISITN)
 )
+```
 
-head(select(advs, USUBJID, PARAMCD, AVISIT, ATPTN, ADT, AVAL, ANL01FL))
+```{r, eval=TRUE, echo=FALSE}
+dataset_vignette(advs, 
+                 display_vars = vars(USUBJID, PARAMCD, AVISIT, ATPTN, ADT, AVAL, ANL01FL),
+                 filter = PARAMCD == 'DIABP' & VISIT %in% c("WEEK 2", "WEEK 8")
+)
 ```
 
 Another common example would be flagging the worst value for a subject,
 parameter, and visit. For this example, we will assume we have 3 `PARAMCD` 
-values (SYSBP, DIABP, and RESP).  We will also assume high is worst for SYSBP 
-and DIABP and low is worst for RESP.
+values (`SYSBP`, `DIABP`, and `RESP`).  We will also assume high is worst for `SYSBP` 
+and `DIABP` and low is worst for `RESP`.
 
 ```{r eval=TRUE}
 advs <- derive_extreme_flag(
```

Comment:
a new function has been added to handle this: `derive_worst_flag()`

Reply:
Done